src/panel.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <title>Block-facebook</title>
+      <link rel="stylesheet" href="panel.css">
+    </head>
+
+    <body>
+      <div id="panel-1" class="wrapper">
+          <h3 class="profile">
+              Facebook Container
+          </h3>
+          <p>
+            Facebook Container has been installed and is preventing Facebook from tracing you around the web. Anytime you see <img src="fb-ban2.png"> this addon is blocking Facebook from traking you.
+          </p>  
+      </div>
+      <div>
+          <button id=2 class="button">Next</button>
+      </div>
+
+      <div id="panel-2" class="wrapper">
+          <h3 class="profile">
+                Facebook Container
+          </h3>
+          <p>
+            On your next visit to Facebook it will be opened in a container, which will isolate your Facebook identy from other websites.
```

Comment:
Check the alignment in this file, I think everywhere else we use 2 spaces indenting.